cex/go-example/main.go

```diff
@@ -7,43 +7,68 @@ import (
 	"fmt"
 	"log"
 	"math/big"
+	"os"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/gofrs/uuid"
+	"github.com/joho/godotenv"
+	"github.com/node-real/megafuel-go-sdk/pkg/paymasterclient"
+	"github.com/node-real/megafuel-go-sdk/pkg/sponsorclient"
 )
 
-const TokenContractAddress = "0x.."
-const WithdrawRecipientAddress = "0x.."
-const SponsorPolicyId = ".."
-const HotwalletPrivateKey = ".."
+var (
+	PAYMASTER_URL string
```

Comment:
As these are variable not const, please dont use `Up case`,  `PaymasterURL` is good enough.

Reply:
fixed